From 72db012c92627593e71a8f303fd6b40548fe7b5d Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Thu, 17 Mar 2022 16:57:35 +0200
Subject: [PATCH] [sw-763872] gaudi2: split ppboot version

since lately we split preboot from ppboot,
Split ppboot version in its,

Change-Id: Iad428de00d2dd4ccd760a2495b90ddd36bb9b641
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/board/habanalabs/gaudi2/agent-fw_loader.its b/board/habanalabs/gaudi2/agent-fw_loader.its
index 8380a62..bd39d99 100644
--- a/board/habanalabs/gaudi2/agent-fw_loader.its
+++ b/board/habanalabs/gaudi2/agent-fw_loader.its
@@ -1,6 +1,6 @@
 /*
  * Simple standalone image source file containing a
- * HabanaLabs Zephyr fw loader agent
+ * reset HabanaLabs Zephyr fw loader agent
  */
 
 /dts-v1/;
@@ -89,3 +89,6 @@
 		};
 	};
 };
+
+
+ddddd
